release/setup.cfg

```diff
@@ -0,0 +1,29 @@
+[metadata]
+name = git-filter-repo
+url = https://github.com/newren/git-filter-repo
+project_urls =
+    Source = https://github.com/newren/git-filter-repo
+    Issues = https://github.com/newren/git-filter-repo/issues/
+description = git filter-branch replacement
+long_description = file: README.md
+author = Elijah Newren
```

Comment:
While I've written virtually everything right now, part of the point of being open source is to allow others to contribute so I'm worried this will go stale.  Is an author field necessary?  What do larger multi-author projects do?

Reply:
You get a warning if you specify nothing -- I *think* the real PyPI may reject the package, but not sure -- the test PyPI doesn't.

In theory there's also `maintainer` instead of `author`.

In practice there are varying approaches, but the main common one is to list something to the effect of "main author and others" and then if need be link to a contributor's file.

E.g. here's what [`coverage.py` does](https://pypi.org/project/coverage/) (in the meta link in the left sidebar).